psqlparse/nodes/value.py

```diff
@@ -35,6 +35,26 @@ def val(self):
         return self.str
 
 
+class Literal(String):
```

Comment:
This file should mimic https://github.com/postgres/postgres/blob/master/src/include/nodes/value.h
The only allowed ttypes are the following: https://github.com/postgres/postgres/blob/master/src/include/nodes/nodes.h#L278-L286